ofl/mochiypopone/METADATA.pb

```diff
@@ -0,0 +1,19 @@
+name: "Mochiy Pop One"
+designer: "FONTDASU"
+license: "OFL"
+category: "HANDWRITING"
```

Comment:
It looks like informal sans to me, it isn't textured or calligraphic, so I recommend sans here

Reply:
Done